transformation-engine/latest/using/index.md

```diff
@@ -12,13 +12,13 @@ Use the Document Transformation Engine Web Console to view information about the
 
 Only Administrators can access and use the Document Transformation Engine Web Console.
```

Comment:
Just a general question: How do I access this page if I don't know the URL?
I cannot find any links to this one.

Reply:
Good spot! I fixed the missing table of contents entry in an [earlier commit](https://github.com/Alfresco/docs-alfresco/pull/453/commits/4cf612e7494bd1e54150dd10e55c47703b9f34d1). You can change the URL in the published docs to `/using/` to find that page: https://docs.alfresco.com/transformation-engine/latest/using/

This will be fixed in both the v2.2 & v2.3 docs once the GA release of DTE 2.3.1 is approved (planned for next week).

Reply:
Hmm .. now I can see what you mean - there are no links to the `/using/` page. **Web Console** is only mentioned in the main landing page: https://docs.alfresco.com/transformation-engine/latest/ 

- Is `/using/index.md` no longer needed?

Reply:
Imho it's not the most useful page since the UI is self-explanatory. So, I wouldn't miss it. 
But we should at least mention somewhere how to find the main page (http://<transformation-host>:<port>/transformation-server/)

Reply:
Thanks Tim - I'll consider this as a future change so as not to delay the current updates.